src/NuGet/VS.ExternalAPIs.Roslyn.Package/VS.ExternalAPIs.Roslyn.Package.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>net472</TargetFramework>
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <!-- This packs up vs debugger config files, which are only generated when using the VS SDK -->
+    <ExcludeFromBuild Condition="'$(UsingToolVSSDK)' != 'true'">true</ExcludeFromBuild>
```

Comment:
Hmm ... what is the `$(UsingToolVSSDK)` property and when is it set / not set?

Reply:
It's set on Windows, not on non-Windows.

Reply:
https://github.com/dotnet/roslyn/blob/main/eng/Versions.props#L64

Reply:
Could we justh change this to an OS check then?

Reply:
Will do